client/lib/common/src/lodash.custom.js

```diff
@@ -1,6 +1,6 @@
 /**
  * @license
- * lodash 3.7.0 (Custom Build) <https://lodash.com/>
+ * lodash 3.8.0 (Custom Build) <https://lodash.com/>
```

Comment:
Actually I don't know, I thought I may have pulled it and got commited after a resolving a conflict or something similar. You didn't add this version?